std-bits/base/src/main/java/org/enso/base/net/URITransformer.java

```diff
@@ -81,4 +80,21 @@ public static String encodeForQuery(String value) {
   public static String encode(String value) {
     return URLEncoder.encode(value, StandardCharsets.UTF_8).replace("+", "%20");
   }
+
+  public static String encodeQuery(String rawQueryString) {
+    var parts = rawQueryString.split("&");
+    StringBuilder sb = new StringBuilder();
+    for (int i = 0; i < parts.length; i++) {
+      if (i > 0) {
+        sb.append("&");
+      }
+      var keyValue = parts[i].split("=");
+      sb.append(encodeForQuery(keyValue[0]));
+      if (keyValue.length > 1) {
+        sb.append("=");
+        sb.append(encodeForQuery(keyValue[1]));
+      }
```

Comment:
What if `keyValue.length > 2`? This discards stuff after a second `=`. I assume such second `=` should be considered to be part of the value and encoded.

Reply:
Or I guess it does not have to be encoded, I think multiple `=` is actually valid and how it is handled depends on the server.

HTTPBin:
![image](https://github.com/enso-org/enso/assets/1436948/c9328d62-3c0c-4f1f-9d60-1709e30c9f5b)